src/utils/testing.tsx

```diff
@@ -211,17 +211,7 @@ export function behavesAsComponent({Component, systemPropArray, toRender, option
 
   const getElement = () => (toRender ? toRender() : <Component />)
 
-  it('implements system props', () => {
-    for (const systemProps of systemPropArray) {
-      expect(Component).toImplementSystemProps(systemProps)
-    }
-  })
-
   if (!options.skipSx) {
-    it('implements the sx prop', () => {
```

Comment:
Do we have another way to make sure that all components are getting the correct system and sx props? Not sure how I feel about removing this

Reply:
I agree that we need a better way to test that system props are being correctly applied but I think that could be done in a separate PR. I don't think that these tests were adding that much value because they weren't testing if passing a system prop applied the correct styles.